mos-platform/common/lib/exit.c

```diff
@@ -0,0 +1,8 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+extern __attribute__((noreturn)) void __exit(int status);
```

Comment:
I've put this attribute here, will it work?

Reply:
Yep!